main/src/ca/uwaterloo/flix/language/errors/SafetyError.scala

```diff
@@ -6,18 +6,21 @@ import ca.uwaterloo.flix.util.vt.VirtualString._
 import ca.uwaterloo.flix.util.vt.VirtualTerminal
 
 /**
-  * A common super-type for safety errors.
-  */
-sealed trait SafetyError extends CompilationError
+ * A common super-type for safety errors.
+ */
+sealed trait SafetyError extends CompilationError {
+  def kind: String = "Safety Error"
+}
 
 object SafetyError {
 
   /**
-    * An error raised to indicate an illegal use of a non-positively bound variable in a negative atom.
-    */
+   * An error raised to indicate an illegal use of a non-positively bound variable in a negative atom.
+   */
   case class IllegalNonPositivelyBoundVariable(sym: Symbol.VarSym, loc: SourceLocation) extends SafetyError {
-    def kind: String = "Safety Error"
+
     def summary: String = s"Illegal non-positively bound variable '$sym'."
+
     def message: VirtualTerminal = {
       val vt = new VirtualTerminal
       vt << Line(kind, source.format) << NewLine
```

Comment:
I guess these changes are not necessary anymore since it was moved to redundancy.